content/master/cli/command-reference.md

```diff
@@ -451,8 +444,7 @@ Using `crossplane xpkg logout` removes the `session` from the
 
 Push a Crossplane package file to a package registry. 
 
-The Crossplane CLI pushes images to the 
-[Upbound Marketplace](https://marketplace.upbound.io/) at `xpkg.upbound.io` by 
+The Crossplane CLI pushes images to the [crossplane-contrib](https://github.com/orgs/crossplane-contrib/packages) GitHub Container Registry at `xpkg.crossplane.io` by 
```

Comment:
this change isn't being done right now, we can maybe make this content more generic (not referencing `xpkg.upbound.io` specifically, and then follow up later on if the CLI code changes.

Reply:
WDYT about just removing the highlighted sentence in entirety for now, so it's just:

>Push a Crossplane package file to a package registry.

Reply:
sounds good to me!